### PR TITLE
Create RU ver of getting_started_github

### PR DESCRIPTION
**Pull request by greggubarev**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [x] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by noroadsleft** [comment]
Requesting reviews from @StrikerRUS and/or @nabokovas.

File: docs/ru-ru/getting_started_github.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
+# Как использовать Github с QMK
```
Comment (StrikerRUS) [inline docs/ru-ru/getting_started_github.md]:
```suggestion
# Как использовать GitHub с QMK
```

File: docs/ru-ru/getting_started_github.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+Github может быть немного сложным для тех, кто не знаком с ним - это руководство будет проходить через каждый шаг разветвления, клонирования и отправки запроса на получение с помощью QMK.
```
Comment (StrikerRUS) [inline docs/ru-ru/getting_started_github.md]: Звучит ужасно, если честно. Надо полностью перефразировать как-то...

File: docs/ru-ru/getting_started_github.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+?> В этом руководстве предполагается, что вы в какой-то степени знакомы с работой в командной строке, и в вашей системе установлен git.
+
+Начните с [страницы QMK Github] (https://github.com/qmk/qmk_firmware), и в правом верхнем углу вы увидите кнопку с надписью «Fork»:
```
Comment (StrikerRUS) [inline docs/ru-ru/getting_started_github.md]:
```suggestion
Откройте [страницу QMK на GitHub] (https://github.com/qmk/qmk_firmware), и в правом верхнем углу вы увидите кнопку с надписью "Fork":
```

File: docs/ru-ru/getting_started_github.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+
+![Fork on Github](http://i.imgur.com/8Toomz4.jpg)
+
+Если вы являетесь частью организации, вам нужно выбрать учетную запись, к которой она будет подключена. В большинстве случаев вы захотите подключить его к своему личному аккаунту. Как только ваш форк завершен (иногда это занимает немного времени), нажмите кнопку «Клонировать или скачать»:
```
Comment (StrikerRUS) [inline docs/ru-ru/getting_started_github.md]:
```suggestion
Если вы состоите в какой-либо организации, вам нужно выбрать учетную запись, к которой будет привязан форк. В большинстве случаев это будет личной аккаунт. Как только ваш форк будет завершен (иногда это занимает немного времени), нажмите кнопку "Clone or Download":
```

File: docs/ru-ru/getting_started_github.md [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+![Download from Github](http://i.imgur.com/N1NYcSz.jpg)
+
+И обязательно выберите «HTTPS», выберите ссылку и скопируйте ее:
```
Comment (StrikerRUS) [inline docs/ru-ru/getting_started_github.md]:
```suggestion
И обязательно выберите "HTTPS", затем выделите ссылку и скопируйте ее:
```

File: docs/ru-ru/getting_started_github.md [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+
+![HTTPS link](http://i.imgur.com/eGO0ohO.jpg)
+
+Отсюда введите `git clone` в командную строку, а затем вставьте ссылку:
```
Comment (StrikerRUS) [inline docs/ru-ru/getting_started_github.md]:
```suggestion
Теперь введите `git clone` в командную строку, а затем вставьте ссылку:
```

File: docs/ru-ru/getting_started_github.md [hunk 7]
```diff
@@ -0,0 +1,58 @@
...
+Cloning into 'qmk_firmware'...
+remote: Counting objects: 46625, done.
+remote: Compressing objects: 100% (2/2), done.
+remote: Total 46625 (delta 0), reused 0 (delta 0), pack-reused 46623
+Receiving objects: 100% (46625/46625), 84.47 MiB | 3.14 MiB/s, done.
+Resolving deltas: 100% (29362/29362), done.
+Checking out files: 100% (2799/2799), done.
+```
+
+Теперь у вас есть форк QMK на вашем локальном компьютере, и вы можете добавить свою таблицу ключей, скомпилировать ее и перенести на свою доску. Как только вы будете довольны своими изменениями, вы можете добавить, зафиксировать и перенести их на свой форк следующим образом:
```
Comment (StrikerRUS) [inline docs/ru-ru/getting_started_github.md]:
```suggestion
Теперь у вас есть форк QMK на вашем локальном компьютере, и вы можете добавить свою раскладку, скомпилировать ее и прошить ей свою клавиатуру. Как только вы будете довольны своими изменениями, вы можете добавить, зафиксировать и перенести их на свой форк следующим образом:
```

Comment (StrikerRUS) [inline docs/ru-ru/getting_started_github.md]:
```suggestion
Теперь у вас есть форк QMK на вашем локальном компьютере, и вы можете добавить свою таблицу ключей, скомпилировать ее и перенести на свою доску. Как только вы будете довольны своими изменениями, вы можете добавить, зафиксировать их и сделать коммит в свой форк следующим образом:
```

Согласно [словарю](https://github.com/web-standards-ru/dictionary/blob/master/git.md).

File: docs/ru-ru/getting_started_github.md [hunk 8]
```diff
@@ -0,0 +1,58 @@
...
+Delta compression using up to 4 threads.
+Compressing objects: 100% (1/1), done.
+Writing objects: 100% (1/1), 1.64 KiB | 0 bytes/s, done.
+Total 1 (delta 1), reused 0 (delta 0)
+remote: Resolving deltas: 100% (1/1), completed with 1 local objects.
+To https://github.com/whoeveryouare/qmk_firmware.git
+ + 20043e64...7da94ac5 master -> master
+```
+
+Ваши изменения теперь существуют на вашей вилке на Github - если вы вернетесь туда (`https://github.com/<whoeveryouare>/qmk_firmware`), Вы можете создать "New Pull Request" нажатием на кнопку:
```
Comment (StrikerRUS) [inline docs/ru-ru/getting_started_github.md]:
```suggestion
Ваши изменения теперь существуют в вашем форке на GitHub - если вы вернетесь туда (`https://github.com/<whoeveryouare>/qmk_firmware`), вы сможете создать "New Pull Request" нажатием на кнопку:
```

File: docs/ru-ru/getting_started_github.md [hunk 9]
```diff
@@ -0,0 +1,58 @@
...
+
+![New Pull Request](http://i.imgur.com/DxMHpJ8.jpg)
+
+Здесь вы сможете увидеть, что именно вы совершили - если все выглядит хорошо, вы можете завершить его, нажав "Create Pull Request":
```
Comment (StrikerRUS) [inline docs/ru-ru/getting_started_github.md]:
```suggestion
Здесь вы сможете увидеть, что именно вы совершили, - если все выглядит хорошо, вы можете завершить его, нажав "Create Pull Request":
```

Три раза `вы` - многовато, надо перефразировать.

File: docs/ru-ru/getting_started_github.md [hunk 10]
```diff
@@ -0,0 +1,58 @@
...
+
+![Create Pull Request](http://i.imgur.com/Ojydlaj.jpg)
+
+После отправки мы можем поговорить с вами о ваших изменениях, попросить вас внести изменения и в конечном итоге принять их! Спасибо за участие в QMK :)
```
Comment (StrikerRUS) [inline docs/ru-ru/getting_started_github.md]:
```suggestion
После отправки мы можем расспросить вас о ваших изменениях, попросить внести корректировки и в конечном итоге принять их! Спасибо за участие в QMK :)
```

Comment (StrikerRUS) [inline docs/ru-ru/getting_started_github.md]:
```suggestion
После отправки мы можем поговорить с вами о ваших изменениях, попросить вас внести изменения и в конечном итоге принять их! Спасибо за ваш вклад в QMK :)
```

File: docs/ru-ru/getting_started_github.md (hunk 2 above)
Comment (greggubarev) [reply docs/ru-ru/getting_started_github.md]: исправил

File: docs/ru-ru/getting_started_github.md [hunk 11]
```diff
@@ -0,0 +1,58 @@
...
+
+Ваши изменения теперь существуют в вашем форке на GitHub - если вернуться туда (`https://github.com/<whoeveryouare>/qmk_firmware`), вы сможете создать "New Pull Request" нажатием на кнопку:
+
+![New Pull Request](http://i.imgur.com/DxMHpJ8.jpg)
+
+Здесь вы сможете увидеть, что именно совершили, - если все выглядит хорошо, вы можете завершить его, нажав "Create Pull Request":
+
+![Create Pull Request](http://i.imgur.com/Ojydlaj.jpg)
+
+После отправки мы можем поговорить с вами о ваших изменениях, попросить вас внести изменения и в конечном итоге принять их! Спасибо за ваш вклад в QMK :)
```
Comment (StrikerRUS) [inline docs/ru-ru/getting_started_github.md]:
```suggestion
После отправки мы можем расспросить вас о ваших изменениях, попросить внести корректировки и в конечном итоге принять их! Спасибо за ваш вклад в QMK :)
```

File: docs/ru-ru/getting_started_github.md [hunk 12]
```diff
@@ -0,0 +1,58 @@
...
+Cloning into 'qmk_firmware'...
+remote: Counting objects: 46625, done.
+remote: Compressing objects: 100% (2/2), done.
+remote: Total 46625 (delta 0), reused 0 (delta 0), pack-reused 46623
+Receiving objects: 100% (46625/46625), 84.47 MiB | 3.14 MiB/s, done.
+Resolving deltas: 100% (29362/29362), done.
+Checking out files: 100% (2799/2799), done.
+```
+
+Теперь у вас есть форк QMK на вашем локальном компьютере, и вы можете добавить свою таблицу ключей, скомпилировать ее и перенести на свою доску. Как только вы будете довольны своими изменениями, есть возможность добавить, зафиксировать их и сделать коммит в свой форк следующим образом:
```
Comment (StrikerRUS) [inline docs/ru-ru/getting_started_github.md]:
```suggestion
Теперь у вас есть форк QMK на вашем локальном компьютере, и вы можете добавить свою раскладку, скомпилировать ее и прошить ей свою клавиатуру. Как только вы будете довольны своими изменениями, есть возможность добавить, зафиксировать их и сделать коммит в свой форк следующим образом:
```

File: docs/ru-ru/getting_started_github.md [hunk 13]
```diff
@@ -0,0 +1,58 @@
+# Как использовать GitHub с QMK
+
+Github может быть немного сложным для тех, кто не знаком с ним - это руководство будет проходить через каждый шаг создания форка, клонирования и отправки пулл реквеста с помощью QMK.
```
Comment (StrikerRUS) [inline docs/ru-ru/getting_started_github.md]:
```suggestion
GitHub может показаться несколько сложным для тех, кто никогда с ним не работал. В данном руководстве будет разобран каждый шаг создания форка, клонирования и отправки пулреквеста в QMK.
```

**Comment by greggubarev** [comment]
> Here are some missed proposals from my previous review and a try to humanize the first sentence.
> 
> Also ping @nabokovas in case you have some sparse time.

@StrikerRUS thanks, edited

File: docs/ru-ru/getting_started_github.md [hunk 14]
```diff
@@ -0,0 +1,58 @@
...
+remote: Resolving deltas: 100% (1/1), completed with 1 local objects.
+To https://github.com/whoeveryouare/qmk_firmware.git
+ + 20043e64...7da94ac5 master -> master
+```
+
+Ваши изменения теперь существуют в вашем форке на GitHub - если вернуться туда (`https://github.com/<whoeveryouare>/qmk_firmware`), вы сможете создать "New Pull Request" нажатием на кнопку:
+
+![New Pull Request](http://i.imgur.com/DxMHpJ8.jpg)
+
+Здесь вы сможете увидеть, что именно совершили, - если все выглядит хорошо, вы можете завершить его, нажав "Create Pull Request":
```
[the remark this answers is not in the document]
Comment (greggubarev) [reply docs/ru-ru/getting_started_github.md]: @nabokovas спасибо, исправил

**Comment by noroadsleft** [comment]
@greggubarev @StrikerRUS @nabokovas Thanks so much for all your efforts on these!

**Pull request merged**